packages/protractor/test/protractor-utils/BUILD.bazel

```diff
@@ -28,5 +28,6 @@ jasmine_node_test(
     srcs = [":protractor_utils_tests_lib"],
     data = [
         ":fake-devserver",
+        "//packages/protractor",
```

Comment:
Why was this one required? Shouldn't that get pulled in with the `srcs`?

Reply:
I think the answer is that the linker needs to see it so the package is resolvable, and the linker doesn't look at the srcs attribute.

Reply:
Actually now that I comment it out, I think you're right that it's not needed, seems to pass the flag flip test without this line. Removed :)